tests/test_harvester_global_prateb.py

```diff
@@ -0,0 +1,86 @@
+import numpy as np
+import os,sys
+from datetime import datetime
+from pathlib import Path 
+import pytest
+import yaml
+import glob
+import xarray as xr
+from netCDF4 import Dataset
+
+from score_hv import hv_registry
+from score_hv.harvester_base import harvest
+from score_hv.yaml_utils import YamlLoader
+from score_hv.harvesters.innov_netcdf import Region, InnovStatsCfg
+
+#TEST_DATA_FILE_NAME = 'bfg_19940101_mean_prateb_control.nc'
```

Comment:
best practice recommendation to not add in code that is commented it out. Ideally it should either be removed if it's not needed or included as a part of the code itself.